File: Practice/src/NAVER2020/Num1.java
package NAVER2020;

public class Num1 {
    public String solution(String m, String k){
        String answer = "";

        int idx = 0;
        for(int i=0;i<k.length();i++){
            char c = k.charAt(i);
            while(idx<m.length()){
                if(m.charAt(idx) == c){
                    m =m.substring(0,idx)+m.substring(idx+1);
                    break;
                }else {
                    idx++;
                }
            }
        }

        return m;
    }
}

File: Practice/src/programmers_challenge/Num1.kt
package programmers_challenge

class Num1 {
    fun solution(numbers: IntArray): IntArray {
        var answer: IntArray = intArrayOf()
        numbers.sort()
        for(i in numbers.indices){
            for(j in i+1 until numbers.size){
                answer = answer.plus(numbers[i]+numbers[j])
            }
        }
        val set= answer.toSet()
        return set.toIntArray().apply { sort() }
    }
}